CONTRIBUTING.md

```diff
@@ -3,19 +3,24 @@
 This document summarizes the most important points for people interested in
 contributing to Redot, especially via bug reports or pull requests.
 
-The [Redot Discord](https://discord.gg/redot) documentation has a dedicated Contributing section
+The [Redot Documentation](https://docs-stable.redotengine.org/contributing/development) has a dedicated Contributing section
```

Comment:
Please only link to https://docs.redotengine.org. The others will be phased out.